requirements.txt

```diff
@@ -5,6 +5,7 @@ boto==2.48.0
 botocore
 certifi
 chardet
+cityhash==0.2.3post9
```

Comment:
Why is this one pinned?

Reply:
See my [prior comment](https://github.com/ska-sa/katsdpdata/pull/161#issuecomment-591368381).

Reply:
Missed that, sorry!